content/tutorials/mushr_navigation_system.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "MuSHR_navigation_system_tutorial"
+date: 2021-04-18T22:17:25+05:30
+summary: "This tutorial covers running the MuSHR multi-agent navigation stack in simulation"
+difficulty: "Advanced"
+duration: 60
+featured: false  # whether this is listed at / (must also be top 6 by weight). 
+active: true     # whether this is listed at /tutorials/
+draft: true      # whether Hugo considers this a draft
+weight: 3        # 2 = intro tutorial 3 = anything else
+---
+
+<h2> By: <a href=https://www.sidharthtalia.com/>Sidharth Talia</a></h2>
+
+<!-- Header figure required! -->
+<br>
+{{< figure src="/tutorials/MuSHR_multiagent_navigation/intro.jpg" width="800" >}} <br>                           
+<br>
+
+### Introduction
+A navigation system enables a robot to navigate quickly between different poses while avoiding collisions with the environment or other agents. The navigation system in this project uses the NH-TTC [backend](https://github.com/davisbo/NHTTC), which is a decentralized multi-agent navigation system. The Non-holonomic-Time-To-Collision (NH-TTC) backend considers the Non-holonomic constraints of the car and considers the time to collision with other cars in the optimization process. The reader may peruse the [paper](http://motion.cs.umn.edu/r/NH-TTC/arxiv-NHTTC.pdf) to better understand how it works.
+
+### Goal
```

Comment:
"the reader" --> you. 

Also isn't the main goal to get it running for the user?

Reply:
yeah. I got confused about that because I was writing the tuning part at the time.